eng/pipelines/templates/jobs/build-validate-pom.yml

```diff
@@ -13,12 +13,18 @@ jobs:
   - job: 'BuildAndValidate'
     displayName: 'Build and Validate'
 
-    variables:
-      - template: /eng/pipelines/templates/variables/globals.yml
+    templateContext:
+      outputs:
+        - output: pipelineArtifact
+          displayName: 'Capture repo state for analysis'
+          targetPath: '$(System.DefaultWorkingDirectory)'
+          artifactName: 'repository'
```

Comment:
This probably works, but `artifactName` here, and you reference it by `$(ArtifactName)` down below. Is there something I'm missing or is that supposed to align?

Reply:
Oh. I see. They're both just uploaded from the parent directory $(Build.ArtifactStagingDirectory) now.

Reply:
Might be a bit off on this one I'll double check